packages/taquito-local-forging/test/codec.spec.ts

```diff
@@ -223,15 +223,15 @@ describe('Tests for Entrypoint functions and for encode and decoder error messag
     expect(() =>
       entrypointDecoder(
         Uint8ArrayConsumer.fromHexString(
-          '085c8244b8de7d57795962c1bfc855d0813f8c61eddf3795f804ccdea3e4c82ae95c8244b8de7d57795962c1bfc855d0813f8c61eddf3795f804ccdea3e4c82ae9'
+          '0a5c8244b8de7d57795962c1bfc855d0813f8c61eddf3795f804ccdea3e4c82ae95c8244b8de7d57795962c1bfc855d0813f8c61eddf3795f804ccdea3e4c82ae9'
```

Comment:
would like to understand the change of starting from 08 to 0a, please

Reply:
This is related to this code in `packages/taquito-local-forging/src/codec.ts`:

```
export const entrypointDecoder = (value: Uint8ArrayConsumer) => {
  const preamble = pad(value.consume(1)[0], 2);

  if (preamble in entrypointMapping) {
    return entrypointMapping[preamble];
  } else {
    const entry = extractRequiredLen(value, 1);

    const entrypoint = Buffer.from(entry).toString('utf8');

    if (entrypoint.length > ENTRYPOINT_MAX_LENGTH) {
      throw new OversizedEntryPointError(entrypoint);
    }
    return entrypoint;
  }
};
```

This test would pass `08` to `preample` which would go to `else` and read the entrypoint name. Because we had added new values to `packages/taquito-local-forging/src/constants.ts`:`entrypointMapping`, this would no longer work. The test would not throw. I passed `0a`, to re-enable the intended failure. We could set a much larger number, like `ff`, but with `0a`, this test is much more likely to fail in case of new constants, which is a good thing.